### DISCUSSION TITLE
Different results between model in OMEdit and its FMU?

### DISCUSSION THREAD

**Discussion by Amuri44** [OpenModelica/OpenModelica#8528] [Q&A]
I have a model R in OpenModelica, in which it is a user-defined block. How to export FMU?

**Comment by casella** [comment]
@arun3688 can you help?

**Reply by arun3688** [reply]
@casella yes i will look into this

**Reply by arun3688** [reply]
@Amuri44 can you run the fmu in `OMSimulator`  and check the results ? otherwise please share the fmu

**Reply by arun3688** [reply]
@Amuri44 ` I run the model in OMEdit over 0:1e-6:32e-3, ` what are these settings you did in `OMEdit` ?

**Reply by arun3688** [reply]
@Amuri44 I am getting the same results like you, i am asking about the tolerance settings or stop time you did in OMEdit which you wrote above `I run the model in OMEdit over 0:1e-6:32e-3,`

**Comment by arun3688** [comment]
@Amuri44 I get the correct results in `OMSimulator`  after changing the parameter values `Pmss=2 and Vfss=2` and simulate in OMEdit. see the image below

![image](https://user-images.githubusercontent.com/6870949/153517544-3d14103b-4524-4ee5-8336-989b4dde6837.png)

You can try to simulate the fmu directly in OMEdit by making an ssp from the OMEdit 

![image](https://user-images.githubusercontent.com/6870949/153518039-444ae49b-291b-4b0c-94eb-dd64cd6dabab.png)

and then select the `type to be strongly connected` and then `add SubModule and load the fmu `

![image](https://user-images.githubusercontent.com/6870949/153519087-18fe1e65-eac4-4cd4-afff-e1f8905b27db.png)

and in the diagram view you can edit the parameters and press the simulate

![image](https://user-images.githubusercontent.com/6870949/153519139-7d9970da-835b-4f2c-8b3f-5c05ce0a0fca.png)
